src/models/predict_lr_model.py

```diff
@@ -0,0 +1,74 @@
+# model neu laden und funktion, wo ein wert eingefügt wird
+# parameter model und ein wert
+# load_model()
+# predict_value()
+
+
+import joblib
+import pandas as pd
+import numpy as np
+
+
+def load_model() -> object:
+    """
+    Load model from disk
+
+    Returns
+    -------
+    object
+        model
+    """
+    model_name = "./models/sklearn_logistic_regression_model.pkl"
+    model = joblib.load(filename=model_name)
+    return model
+
+
+def predict_probability(model: object, value: str, category_list: list) -> pd.DataFrame:
```

Comment:
Hier muss auch object auch wieder zu imblearn.pipeline.Pipeline
Und am besten auch angeben was für werte in list übergeben werden, also List[str] (https://stackoverflow.com/questions/24853923/type-hinting-a-collection-of-a-specified-type)